src/fastoad/io/configuration/resources/configuration.json

```
@@ -134,9 +134,43 @@
       "type": "string",
       "default": "./outputs.xml"
     },
+    "sys_paths": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "imports": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
     "driver": {
-      "type": "string",
-      "default": "om.ScipyOptimizeDriver(tol=1e-6, optimizer='SLSQP')"
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "Old syntax for driver configuration"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "instance": {
+              "type": "string",
+              "description": "Driver instance string"
+            },
+            "options": {
+              "type": "object",
+              "description": "Driver options",
+              "additionalProperties": {
+                "type": ["string", "number", "boolean"]
+              }
+            }
+          },
+          "required": ["instance"],
+          "additionalProperties": false
```

You need to allow additional properties to have the `advanced_options` working as in the documentation.
```suggestion
          "additionalProperties": true
```

---

Corrected in [b907e10](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/b907e10574bbea0ccc9570d05088781856e0e8cd)